apps/codeforafrica/src/lib/index.js

```diff
@@ -158,9 +159,11 @@ async function getProcessedNewsAndStories() {
 }
 
 async function getHomePageStaticProps() {
+  const seo = getSeo("index");
   return {
     props: {
       title: "Code for Africa",
```

Comment:
Remove **all** hard-coded titles

Reply:
@kilemensi how would we then get the Seo for specific pages, say About?

Reply:
I think this comment is supposed to be for the `title` in the returned `props` @kelvinkipruto.

Reply:
ooh, Ok. It is much clearer now.